### PR TITLE
swift-module-digester: include operator overloads in the module dump.

### PR DESCRIPTION
**Pull request by nkcsgexi**

### CODE REVIEW

**Comment by nkcsgexi** [comment]
@swift-ci please smoke test

**Comment by nkcsgexi** [comment]
@swift-ci Please smoke test Linux platform

File: test/api-digester/source-stability.swift.expected [hunk 1]
```diff
@@ -119,7 +119,43 @@ AssociatedType StringProtocol.UTF16View has been added as a protocol requirement
...
+Func BinaryInteger.%(_:_:) has been added as a protocol requirement
+Func BinaryInteger.%=(_:_:) has been added as a protocol requirement
+Func BinaryInteger.&(_:_:) has been added as a protocol requirement
+Func BinaryInteger.&=(_:_:) has been added as a protocol requirement
+Func BinaryInteger.<<(_:_:) has been added as a protocol requirement
+Func BinaryInteger.<<=(_:_:) has been added as a protocol requirement
+Func BinaryInteger.>>(_:_:) has been added as a protocol requirement
+Func BinaryInteger.>>=(_:_:) has been added as a protocol requirement
+Func BinaryInteger.\/(_:_:) has been added as a protocol requirement
+Func BinaryInteger.\/=(_:_:) has been added as a protocol requirement
```
Comment (jrose-apple) [inline test/api-digester/source-stability.swift.expected]: This looks like something got escaped improperly.

**Comment by jrose-apple** [comment]
I'm kind of terrified that these weren't being checked before.

**Comment by nkcsgexi** [comment]
We were not checking them for api stability, but we will when we start to generate baseline for abi stability and check it.

**Pull request merged**